src/webgui/src/webgui/image_converter.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from std_msgs.msg import String
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge, CvBridgeError
+
+import roslib
+import sys
+import rospy
+import cv2
+
+roslib.load_manifest('webgui')
```

Comment:
Looks like this is no longer needed

https://answers.ros.org/question/115346/what-does-roslibload_manifest-do/